src/Plugin/Field/FieldType/OgMembershipReferenceItemList.php

```diff
@@ -75,15 +76,15 @@ public function postSave($update) {
     // todo: move to API function.
     $membership_ids = \Drupal::entityQuery('og_membership')
       ->condition('uid', $this->getEntity()->id())
-      ->condition('entity_type', $this->getFieldDefinition()->getTargetEntityTypeId())
+      ->condition('entity_type', $this->getFieldDefinition()->getFieldStorageDefinition()->getSetting('target_type'))
```

Comment:
This bug caused creation of multiple OgMemberships. Maybe worth adding some duplication prevention in preSave()